overlord/snapstate/backend.go

```diff
@@ -79,3 +92,74 @@ func (s *defaultBackend) Download(name, channel string, meter progress.Meter) (s
 
 	return downloadedSnapFile, snap.Developer(), nil
 }
+
+func (s *defaultBackend) CheckSnap(snapFilePath string, flags snappy.InstallFlags) error {
+	meter := &progress.NullProgress{}
+	return snappy.CheckSnap(snapFilePath, flags, meter)
+}
+
+func (s *defaultBackend) SetupSnap(snapFilePath string, flags snappy.InstallFlags) (string, error) {
+	meter := &progress.NullProgress{}
+	return snappy.SetupSnap(snapFilePath, flags, meter)
+}
+
+func (s *defaultBackend) CopySnapData(snapInstPath string, flags snappy.InstallFlags) error {
+	sn, err := snappy.NewInstalledSnap(filepath.Join(snapInstPath, "meta", "snap.yaml"))
+	if err != nil {
+		return err
+	}
+	return snappy.CopyData(sn, flags, &progress.NullProgress{})
```

Comment:
2 of these new methods instantiate the `meter` inline to the call to `snappy.XXX` but the other 4 instantiate it before the call. Consistency would be nice.